x/execution/internal/keeper/keeper.go

```diff
@@ -469,3 +406,53 @@ func (k *Keeper) fetchEmitters(ctx sdk.Context, proc *process.Process, nodeKey s
 	}
 	return matchedRuns, nil
 }
+
+func (k *Keeper) calculatePrice(ctx sdk.Context, exec *executionpb.Execution) (sdk.Int, error) {
+	inst, err := k.instanceKeeper.Get(ctx, exec.InstanceHash)
+	if err != nil {
+		return sdk.Int{}, err
+	}
+	srv, err := k.serviceKeeper.Get(ctx, inst.ServiceHash)
+	if err != nil {
+		return sdk.Int{}, err
+	}
+	task, err := srv.GetTask(exec.TaskKey)
+	if err != nil {
+		return sdk.Int{}, err
+	}
+	perCall := sdk.NewInt(200)
```

Comment:
these constants will move into module parameter in another PR

Reply:
moving this to todo